script/build.sh

```diff
@@ -61,10 +61,21 @@ cp -r $DIR/docs ${SRC_DIR}/pfe/portal/
 echo -e "\n+++   DOWNLOADING EXTENSIONS   +++\n";
 mkdir -p ${SRC_DIR}/pfe/extensions
 rm -f ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-*.zip
+# In case of ARCH = ppc64le, do not download the zip file.
+if [ "$ARCH" == "ppc64le" ]; then
+   echo -e "\n+++++   DISABLING APPSODY EXTENSION FOR PPC64LE ARCHITECTURE +++++\n";
+else
+   curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-9.9.9999.zip http://download.eclipse.org/codewind/codewind-appsody-extension/master/latest/codewind-appsody-extension-9.9.9999.zip
+fi
+
 rm -f ${SRC_DIR}/pfe/extensions/codewind-odo-extension-*.zip
-curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-9.9.9999.zip http://download.eclipse.org/codewind/codewind-appsody-extension/master/latest/codewind-appsody-extension-9.9.9999.zip
 curl -Lo ${SRC_DIR}/pfe/extensions/codewind-odo-extension-9.9.9999.zip http://download.eclipse.org/codewind/codewind-odo-extension/master/latest/codewind-odo-extension-9.9.9999.zip
 
+# In case of ppc64le, download 'appsody' using ppc64le specific url and build 'appsody-controller' from source
+if [ "$ARCH" == "ppc64le" ]; then
+  $DIR/script/build-appsody-controller.sh
```

Comment:
What does this do?  This PR does not include this script.

Reply:
removed.